src/utils/goog.js

```diff
@@ -313,6 +315,9 @@ const replaceLegacyRequire = (root, toReplace) => {
     }
   }
 
+  // add .getInstance() if needed for the in-line replacements
+  if (singleton === true) varName = `${varName}.getInstance()`;
```

Comment:
This is not the proper way to form the AST node for what you're doing here. The MemberExpression should be replaced with an ExpressionStatement that calls `getInstance` from `varName`. You can see the AST tree if you go to [ASTExplorer](https://astexplorer.net/), change the parser to `esprima`, and put this as the source:

```
alertManager.getInstance()
```

Reply:
Booya -- now I'm using the memberexpressions so it looks right in the AST.